ethsigner/signer/hashicorp/src/main/java/tech/pegasys/ethsigner/signer/hashicorp/HashicorpSubCommand.java

```diff
@@ -75,10 +76,28 @@
       arity = "1")
   private String signingKeyPath = DEFAULT_KEY_PATH;
 
+  @Option(
+      names = {"--no-tls-enabled"},
+      negatable = true,
+      defaultValue = "true",
+      description = "Connect to Hashicorp Vault server using TLS (default: ${DEFAULT-VALUE})",
+      required = true)
```

Comment:
The "boolean" has different semantics in PicoCLI i.e. it doesn't take a value, only "option" name can be specified to "enable" it. Since we want TLS for hashicorp to be enabled by default, I would need to rename the variable to "no-tls" instead to disable the TLS (as the absence would implies TLS is enabled). The negatable makes is more user friendly (albeit not code-friendly). `tls-enabled` and `no-tls-enabled`. https://picocli.info/#_boolean_options ...

Reply:
This one is updated to use --tls-enabled=false i.e. boolean option.